ci_framework/roles/build_containers/molecule/hotfix/converge.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_build_containers_cleanup: true
+    cifmw_build_containers_run_hotfix: true
+    cifmw_build_containers_hotfix_tag: patched
+    cifmw_build_containers_hotfix_images:
+      - quay.io/podified-main-centos9/openstack-keystone:current-podified
```

Comment:
it should be `quay.io/podified-master-centos9/openstack-keystone:current-podified`